src/overlays/actors/ovl_En_Invadepoh_Demo/z_en_invadepoh_demo.c

```diff
@@ -54,115 +124,688 @@ static InitChainEntry D_80C1AA88[] = {
+    // EnInvadepoh. Similarly, this function requires a lot of extra stack space for unused variables; the names and
+    // types of these variables are inferred from EnInvadepoh's version of this code.
+    if (true) {
+        Gfx* gfx;
+        Vec3f glowOffset;
+        Vec3f glowPos; // unused, inferred from EnInvadepoh
+        s32 glowAlpha; // unused, inferred from EnInvadepoh
+
+        OPEN_DISPS(play->state.gfxCtx);
+
+        gfx = POLY_XLU_DISP;
+        gfx = Gfx_SetupDL20_NoCD(gfx);
+        gSPSetOtherMode(gfx++, G_SETOTHERMODE_H, 4, 4, 0x00000080);
+        gDPSetCombineLERP(gfx++, 0, 0, 0, PRIMITIVE, TEXEL0, 0, PRIMITIVE, 0, 0, 0, 0, PRIMITIVE, TEXEL0, 0, PRIMITIVE,
+                          0);
+        Matrix_Mult(&play->billboardMtxF, MTXMODE_NEW);
+        Matrix_MultVecZ(90.0f, &glowOffset);
+        Matrix_Translate(this->actor.world.pos.x + glowOffset.x, this->actor.world.pos.y + glowOffset.y + 25.0f,
+                         this->actor.world.pos.z + glowOffset.z, MTXMODE_NEW);
+        Matrix_Scale(0.15f, 0.15f, 0.15f, MTXMODE_APPLY);
+        gSPDisplayList(gfx++, gameplay_keep_DL_029CB0);
+        gDPSetPrimColor(gfx++, 0, 0, 240, 180, 100, 100);
+
+        mtx = Matrix_NewMtx(play->state.gfxCtx);
+
+        if (mtx != NULL) {
+            gSPMatrix(gfx++, mtx, G_MTX_NOPUSH | G_MTX_LOAD | G_MTX_MODELVIEW);
+            gSPDisplayList(gfx++, gameplay_keep_DL_029CF0);
+            POLY_XLU_DISP = gfx;
+        }
+
+        CLOSE_DISPS(play->state.gfxCtx);
+    }
+
+    Matrix_Pop();
+}
+
+void EnInvadepohDemo_Romani_Draw(EnInvadepohDemo* this, PlayState* play) {
+    s32 pad[2];
+
+    OPEN_DISPS(play->state.gfxCtx);
+
+    Gfx_SetupDL25_Opa(play->state.gfxCtx);
+    gSPSegment(POLY_OPA_DISP++, 0x08, Lib_SegmentedToVirtual(&gRomaniEyeSadTex));
+    gSPSegment(POLY_OPA_DISP++, 0x09, Lib_SegmentedToVirtual(&gRomaniMouthHangingOpenTex));
+    SkelAnime_DrawFlexOpa(play, this->skelAnime.skeleton, this->skelAnime.jointTable, this->skelAnime.dListCount, NULL,
+                          NULL, &this->actor);
+
+    CLOSE_DISPS(play->state.gfxCtx);
+}
+
+s32 EnInvadepohDemo_Cow_OverrideLimbDraw(PlayState* play, s32 limbIndex, Gfx** dList, Vec3f* pos, Vec3s* rot,
+                                         Actor* thisx) {
+    if (limbIndex == COW_LIMB_NOSE_RING) {
+        *dList = NULL;
+    }
+
+    return false;
+}
+
+void EnInvadepohDemo_Cow_Draw(EnInvadepohDemo* this, PlayState* play) {
+    Gfx_SetupDL37_Opa(play->state.gfxCtx);
+    SkelAnime_DrawFlexOpa(play, this->skelAnime.skeleton, this->skelAnime.jointTable, this->skelAnime.dListCount,
+                          EnInvadepohDemo_Cow_OverrideLimbDraw, NULL, &this->actor);
+}
+
+void EnInvadepohDemo_Ufo_Draw(EnInvadepohDemo* this, PlayState* play) {
+    s32 pad[2];
+    Vec3f flashPos = gZeroVec3f;
+    Mtx* mtx;
+
+    flashPos.x = this->actor.world.pos.x;
+    flashPos.y = this->actor.world.pos.y;
+    flashPos.z = this->actor.world.pos.z;
```

Comment:
nice wasted cycles lol

Reply:
Yeah, in EnInvadepoh, it looks like this:
```c
    Matrix_Push();
    Matrix_Mult(&play->billboardMtxF, MTXMODE_NEW);
    Matrix_MultVecZ(200.0f, &flashPos);
    Matrix_Pop();
    flashPos.x += this->actor.world.pos.x;
    flashPos.y += this->actor.world.pos.y;
    flashPos.z += this->actor.world.pos.z;
```
And I think they just copy-pasted that and trimmed out the bits they didn't need until it looked something like this lol.